cloudinit/net/dhcp.py

```diff
@@ -288,17 +311,22 @@ def parse_static_routes(rfc3442):
               first parsing error.
 
     E.g.
-    sr = parse_state_routes("32,169,254,169,254,130,56,248,255,0,130,56,240,1")
-    sr = [
+    sr=parse_static_routes("32,169,254,169,254,130,56,248,255,0,130,56,240,1")
+    sr=[
```

Comment:
why this  whitespace change?

Reply:
more questionably ... why did that pass pycodestyle ?

Reply:
@smoser because it's not an actual line of code, it's a comment.  

The reasoning for removing the whitespace is that I was fixing a typo in the comment (the function name see discussion above by ghislain) which added one character to the line.  That character of extra did fix the typo but went over 80 characters which failed the pep8 check.  As this isn't real code but an example, I weighted that it's better to have it be on one line for clarities sake (and aligned the line below).

I didn't think too much over it as it's a comment and I think it's clearer that way than trying to respect Pep8 over a comment that isn't that relevant.

Reply:
Let me know if something is preventing you to resolve this discussion :).  I'm open to discuss this (if it wasn't already clear).

Reply:
@igalic ?

Reply:
i don't have merge rights so my opinions don't count for much 😅

Reply:
just resolving your comment does much :)

Reply:
Oh wait, are you maybe not seeing resolve conversation on your side?

Reply:
then do you consider you comment to be resolved?